### PR TITLE
Fix `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by silverhook**
## Prerequisites

- [x] My Pull Request is filled against the `next` branch
- [ ] I have updated the [CHANGELOG](https://github.com/Pelican-Elegant/elegant/blob/next/CHANGELOG.md) ↦ no, because it’s a simple documentation chore

## Recommended Steps

- [ ] My patch adds a new feature, therefore I have also added a help article about it
- [ ] My patch changes Elegant behavior, therefore I have updated the help article to reflect this change
- [ ] My commits are [signed](https://help.github.com/en/articles/signing-commits)

## Description

This is a partial solution to https://github.com/Pelican-Elegant/elegant/issues/449

### CODE REVIEW

**Comment by talha131** [comment]
@silverhook thank you for your contribution. Your commit messages does not follow the guideline. 

I can either fix them myself or if you can wait, I will add the detailed article on it by Saturday which you can then use to fix your commits.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-See our documentation at following link
+See our documentation at following link:
 
-http://localhost:9000/categories.html#contributing-ref
+<https://elegant.oncrashreboot.com/categories#contributing-ref>
```
[the remark this answers is not in the document]
Comment (silverhook) [reply CONTRIBUTING.md]: Following MarkDown (or rather its actually standard CommonMark) spec, all standalone URLs should be enclosed in them.

See:
https://spec.commonmark.org/0.29/#autolink

**Comment by silverhook** [comment]
>    Moving files into folders is not enough for changing the category. You will have to change the category too

It should be enough to have just one or the other, having both seems to cause needless confusion and duplication. In that case I would suggest we simply strip the category field from the posts and categorise by folders, as this would be much easier to manage when the articles start piling up.

>    "Deployment and CD" What is CD?
>    "Deployment and CD" is not necessary. Moreover the tips shared in these articles are not being used in the project anymore. I think "Development" or "Behind the scenes" is fine.

CD = Continuous Deployment (i.e. the “CD” in CI/CD)

I’m confused with the Development category. The articles there all seemed to be about how to deploy (in a CI/CD manner) Pelican and Elegant, not about how to contribute to the project (which has its own category) or develop it further locally. In the end we could also just have “Miscellaneous” as a catch-all for whatever we can’t categorise otherwise.

> If you disagree about categories then perhaps you can move these categories commits to a new PR. Because rest of your commits are fine except one input.

OK, feel free to drop the one you disagree with and I’ll work on that in a separate PR.

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (talha131) [reply CONTRIBUTING.md]: Thank you. I didn't know about it.

**Comment by talha131** [comment]
@silverhook I have merged all your commits except categories one. I am closing this PR because there is nothing else left to do.

As for your questions, I am going to respond to them here https://github.com/Pelican-Elegant/elegant/issues/314, in order to consolidate our discussion on categories.

**Comment by silverhook** [comment]
Perfect, thanks. Sorry again for the mess…

**Comment by talha131** [comment]
No issues. No need to apologize.

**Pull request closed without merging**